src/Files.App/Helpers/Win32/Win32PInvoke.Consts.cs

```diff
@@ -45,8 +40,18 @@ public static partial class Win32PInvoke
 		public const uint OPEN_ALWAYS = 4;
 		public const uint TRUNCATE_EXISTING = 5;
 
-		public const int MAXIMUM_REPARSE_DATA_BUFFER_SIZE = 16 * 1024;
+		// FSCTL
 		public const int FSCTL_GET_REPARSE_POINT = 0x000900A8;
+		public const int FSCTL_LOCK_VOLUME = 0x00090018;
+		public const int FSCTL_DISMOUNT_VOLUME = 0x00090020;
+		public const uint FSCTL_SET_COMPRESSION = 0x9C040u;
+
+		// Compression
+		public const ushort COMPRESSION_FORMAT_NONE = 0x0000;
+		public const ushort COMPRESSION_FORMAT_DEFAULT = 0x0001;
```

Comment:
You can use COMPRESSION_FORMAT enum.

Reply:
This should be resolved with 31289160e0e62cb693cbe5d85fc6026355b58fc7